dask/dataframe/io/tests/test_parquet.py

```diff
@@ -4403,6 +4403,43 @@ def test_retries_on_remote_filesystem(tmpdir):
         assert layer.annotations["retries"] == 2
 
 
+def test_filesystem_option(tmpdir, engine):
```

Comment:
Maybe parametrize this to test `"fsspec"` and `None` similar to `test_pyarrow_filesystem_option`?